docs/src/pages/demos/autocomplete/IntegrationDownshiftExtra.js

```diff
@@ -0,0 +1,190 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import deburr from 'lodash/deburr';
+import keycode from 'keycode';
+import Downshift from 'downshift';
+import { withStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import Chip from '@material-ui/core/Chip';
+import { suggestions, styles, renderInput, renderSuggestion } from './IntegrationDownshift.common';
+
+function getSuggestions(value) {
+  const inputValue = deburr(value.trim()).toLowerCase();
+  let count = 0;
+
+  // search for all string instead of
+  // only the beginning
+  return suggestions.filter(suggestion => {
+    const keep = count < 5 && suggestion.label.toLowerCase().includes(inputValue);
+
+    if (keep) {
+      count += 1;
+    }
+
+    return keep;
+  });
+}
+
+class DownshiftMultiple extends React.Component {
```

Comment:
Kept this class here in case we want to implement the demo with multiple select also.